### ISSUE TITLE
How to set unselectedTitleColor?

### ISSUE THREAD

**Issue by SrdjanSpasojevic** [subinspathilettu/SJSegmentedViewController#125]
I see there is a variable named `selectedSegmentTitleColor` but I cannot set the color. Is there a way to do that?

**Comment by subinspathilettu** [comment]
@SrdjanSpasojevic you want to change the unselected segment title color?

**Comment by Md-Rais** [comment]
Yes I want to set selected segment title color. But there is no such property to change the selected segment title color
